packages/wonder-blocks-accordion/src/components/__tests__/accordion.test.tsx

```diff
@@ -359,6 +359,32 @@ describe("Accordion", () => {
         });
     });
 
+    test("prioritizes the parent's includeAnimation prop", () => {
+        // Arrange
+        render(
+            <Accordion includeAnimation={true}>
+                {[
+                    <AccordionSection
+                        header="Title"
+                        includeAnimation={false}
+                        headerTestId="section-header-test-id"
+                    >
+                        Section content
+                    </AccordionSection>,
+                ]}
+            </Accordion>,
+            {wrapper: RenderStateRoot},
+        );
+
+        // Act
+        const sectionHeader = screen.getByTestId("section-header-test-id");
+
+        // Assert
+        expect(sectionHeader).toHaveStyle({
+            transition: "border-radius 300ms",
+        });
```

Comment:
**suggestion (non-blocking):** Could you leave a comment about why this assertion verifies what the test's title says it's testing?